common/leica_sep.c

```diff
@@ -53,7 +53,7 @@ static int leica_sep_transfer(struct leica_sep_funcs *f, char *cmd,
 			continue;
 
 		if (size <= bytes_received)
-			return -1;
+			break;
```

Comment:
Isn't it the same as before?
If we break, here https://github.com/hexagon-geo-surv/u-boot-imx/blob/b6bab515d01eb63e14e639fc49d6c3694e2be753/common/leica_sep.c#L96 will return -1, since the command is not yet received

Reply:
yes - but we have only one exit point for errors